iceoryx_hoofs/test/moduletests/test_relative_pointer.cpp

```diff
@@ -194,7 +181,7 @@ TYPED_TEST(RelativePointer_test, ConstrTests)
         // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-pointer-arithmetic, cppcoreguidelines-pro-type-reinterpret-cast)
         auto* typedPtr = reinterpret_cast<TypeParam*>(ptr1 + offset);
         RelativePointer<TypeParam> rp(typedPtr);
-        EXPECT_NE(rp, nullptr);
+        EXPECT_TRUE(rp);
```

Comment:
This is probably required now that implicit conversion is disabled. I wonder why it worked in the first place (without implicit conversion).

Reply:
Good question.. :thinking: